core/src/main/resources/messages_en.properties

```diff
@@ -42,6 +42,15 @@ sincerely.yours.greenCity=Sincerely yours, Green City team.
 sincerely.yours.Ubs=Sincerely yours, Pick Up City team.
 unsubscribe.text=If you no longer wish to receive these emails, you can
 unsubscribe=unsubscribe
+unlock=Unlock account
+block.account=Account blocked
+text.account.ban=Your account has been locked for security reasons due to a possible hacking attempt. For your safety, we recommend changing your password after restoring your account.
+warning=Warning!
+block.user=Your account is blocked
```

Comment:
why do we need `Account blocked` and `Your account is blocked`?

Reply:
'Account blocked' was deleted